Neos.Cache/Classes/Psr/Cache/CachePool.php

```diff
@@ -0,0 +1,176 @@
+<?php
+namespace Neos\Cache\Psr\Cache;
+
+/*
+ * This file is part of the Neos.Cache package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Cache\Frontend\VariableFrontend;
+use Neos\Cache\Psr\InvalidArgumentException;
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+
+/**
+ * A frontend that implements the CacheItemPoolInterface from the PSR-6 specification.
+ */
+class CachePool extends VariableFrontend implements CacheItemPoolInterface
+{
```

Comment:
Not sure why this extends the VariableFrontend other than sharing the cacheFactory. Imho it would be much clearer to have a separate CachePoolFactory as for the simpleCache.

Reply:
I wanted to have this implement the Flow CacheInterface as well, so you can configure it like a frontend. So PSR-6  caches can be instantiated via CacheManager and Objects configuration.
Of course we can change that. Would like to hear more opinions.

Reply:
Imho this sounds hacky. I would try to achieve that with two separate getPsr*Cache methods in the cacheManager. That way you can inject a PSR-cache via objects.yaml and still use the backendConfiguration from Caches.yaml. If another class still uses the another Cache-Frontend it can get this injected as well and we end up having two frontends that share a single backend.